src/container/displayBook.js

```diff
@@ -19,27 +18,44 @@ useEffect(() => {
   apiBooks();
 }, []);
 
+const randomNum = Math.floor(Math.random() * 100)
+const barstyle =  {
+    width: `${randomNum}%`
+};
```

Comment:
**Changes Required**
- [ ] Nice try using the barstyle. However, the design from Zeplin uses circular progress indictors for completion as shown below. Following design specification is one of the requirements you will encounter as a front-end engineer. Let's try to follow the design specification even though as  placeholders.
**Your Design**
![stylebar](https://user-images.githubusercontent.com/7220167/131221936-ab1f931a-c127-456e-bc09-03fc67e66499.png)
**Zeplin Design**
![zaplin-progressive](https://user-images.githubusercontent.com/7220167/131221957-2cb75eb2-e669-40f9-b272-75ad160f2699.png)